### PR TITLE
[Changelog] Add note re php-curl dependency now required for dev install

### LINKED ISSUE #6509: [Changelog] Add note re php-curl dependency now required for dev install
Given `php7.3-curl` is now required for dev installs since #6187 --
This should be noted in the Changelog so that it makes it into the 23 Release Notes under the `Notes for Existing Projects` section

@laemtl -- we've just started the changelog for this release so let's see how the PR discussion goes with the team for this kind of change...

### PR DESCRIPTION
**Pull request by laemtl**
I added a note in Changelog under the Notes for Existing Projects section for the new  php7.3-curl dev install dependencies. I also fixed a redundancy in the file.

- [x] Have you updated related documentation?

* Resolves #6509

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -74,6 +68,7 @@ exception). It is recommended to run this tool for existing projects (PR #5270)
 
 ### Notes For Existing Projects
 - PHP should be upgraded to 7.3 to before upgrading LORIS.
+- For dev instances, php7.3-curl is now a required dependency.
```
Comment (ridz1208) [inline CHANGELOG.md:71]: @AlexandraLivadas why only for DEV ?

Comment (christinerogers) [reply CHANGELOG.md:71]: I think you mean Laetitia @laemtl 

@johnsaigle's PR header in #6187 suggests this is only an issue on Dev environments. 
John was also [asked for clarification on this exact point](https://github.com/aces/Loris/issues/6473#issuecomment-624948602) but not response so far... 

@ridz1208 Are you confirming that it applies to production too or just asking?

Comment (ridz1208) [reply CHANGELOG.md:71]: from what I can see it is indeed just a dev thing. I think it's still worth properly testing however

Comment (christinerogers) [reply CHANGELOG.md:71]: 👍 thanks @ridz1208 this might need shen's help with the install VMs

Comment (kongtiaowang) [reply CHANGELOG.md:71]: @ridz1208 @christinerogers This is correct, php-webdriver requires the php7.3-curl in the dev environment.

Comment (johnsaigle) [reply CHANGELOG.md:71]: @ridz1208 It's definitely only a dev requirement. https://github.com/aces/Loris/pull/6187 updates a package that was renamed. This package requires php7.X-curl. That package drives our integration tests. If you're not running integration tests, the package and its dependency are not required. It's only installed when someone runs `make dev`.

Comment (christinerogers) [reply CHANGELOG.md:71]: thanks for confirming @johnsaigle

**Comment by christinerogers** [comment]
Looks like this is ready to merge for 23 @driusan 

Thanks @laemtl

**Pull request merged**